example/main.go

```diff
@@ -135,6 +136,20 @@ func resolveSecretReference(client *onepassword.Client, vaultID, itemID, fieldID
 	// [developer-docs.sdk.go.resolve-secret]-end
 }
 
+func resolveBulkSecretReferences(client *onepassword.Client, vaultID, itemID, fieldID, fieldID2 string) {
+	// [developer-docs.sdk.go.resolve-bulk-secret]-start
+	// Retrieves a secret from 1Password.
+	// Takes a secret reference as input and returns the secret to which it points.
+	secret, _ := client.Secrets().ResolveAll(context.Background(), []string{fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID), fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID2)})
```

Comment:
`[nitpick]`
This is a very long line and as a user I can easily get lost in it. Would it help to chop it into multiple lines.

Here's a suggestion:
```suggestion
	secret, _ := client.Secrets().ResolveAll(
		context.Background(), 
		[]string{
			fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID), 
			fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID2),
		},
	)
```

Another suggestion here is to export the format of the reference into a variable.